src/extras-openmoji/climate-environment/E2C0.svg

```diff
@@ -0,0 +1,20 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Too big, don't need the white background for the sky, only the white shape for the iceberg, the shape of the water should be considered - maybe it should be the shape of the circle of the grid? 
![image](https://user-images.githubusercontent.com/6676843/74882454-1e055d80-533d-11ea-952b-5165ab23f3da.png)

Reply:
also famously shouldn't there only be a tiny "tip" of the iceberg that's much smaller than the rest? I.E. it should be much more submerged

Reply:
Thanks! Good ideas! Will make a tiny tip and try to make it a bit smaller ... however note we do have really "big" emoijs e.g. https://openmoji.org/library/#search=evening&emoji=1F306